podman_compose.py

```diff
@@ -480,6 +484,19 @@ def get_mount_args(compose, cnt, volume):
     basedir = compose.dirname
     if is_str(volume): volume = parse_short_mount(volume, basedir)
     mount_type = volume["type"]
+
+    vol = None
+    source = volume.get('source')
+    if source:
+      vol = compose.shared_vols[source]
```

Comment:
For `{'type': 'bind', ...}` this throws a KeyError.
I wrapped this whole block in a `if mount_type == 'volume':`. Not sure though if it has other side effects.